master/pkg/searcher/asha.go

```diff
@@ -40,30 +41,31 @@ type (
 
 	// rung describes a set of trials that are to be trained for the same number of units.
 	rung struct {
-		UnitsNeeded   model.Length  `json:"units_needed"`
-		Metrics       []trialMetric `json:"metrics"`
-		StartTrials   int           `json:"start_trials"`
-		PromoteTrials int           `json:"promote_trials"`
+		UnitsNeeded   expconf.Length `json:"units_needed"`
```

Comment:
do `model.Length` and `expconf.Length` serialize to and from the same json repr? (rungs are in the state..)